x/ibc/07-tendermint/types/update.go

```diff
@@ -125,13 +138,23 @@ func checkValidity(
 		Header: &trustedHeader,
 	}
 
+	chainID := clientState.GetChainID()
+	// If chainID is in epoch format, then set epoch number of chainID with the epoch number
+	// of the header we are verifying
+	// This is useful if the update is for a previous epoch then the latest epoch of the client,
+	// then the chainID can be set correctly for the previous epoch before verifying
+	// Updates for previous epochs are not supported if the chainID is not in epoch format
```

Comment:
```suggestion
	// This is useful if the update is for a previous epoch rather than an update for the latest 
	// epoch of the client. The chainID must be set correctly for the previous epoch before attempting
	// verification. Updates for previous epochs are not supported if the chainID is not in epoch format.
```

Reply:
Edited to keep each comment a sentence, rather than splitting sentences across multiple lines. It's more readable for me that way personally